repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/services/datastore/RemoteDatastoreServiceTest.groovy

```diff
@@ -101,7 +101,7 @@ class RemoteDatastoreServiceTest extends Specification {
     }
 
     static def waitUntilReadyToServiceRequests(ReposeLogSearch reposeLogSearch) {
-        reposeLogSearch.awaitByString("Repose ready", 1, 60, TimeUnit.SECONDS)
+        reposeLogSearch.awaitByString("Repose ready", 1, 120, TimeUnit.SECONDS)
```

Comment:
Do we want to pull the `120` from somewhere instead of specifying the literal here?

Reply:
On it.